profiles/schema.py

```diff
@@ -0,0 +1,131 @@
+import graphene
+from django.conf import settings
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required
+from munigeo.models import AdministrativeDivision
+from thesaurus.models import Concept
+
+from .models import Profile
+
+
+class ConceptType(DjangoObjectType):
+    class Meta:
+        model = Concept
+        fields = ("code",)
+
+    vocabulary = graphene.String()
+    label = graphene.String()
+
+    def resolve_vocabulary(self, info, **kwargs):
+        return self.vocabulary.prefix
+
+
+class AdministrativeDivisionType(DjangoObjectType):
+    class Meta:
+        model = AdministrativeDivision
+        fields = ("children", "origin_id", "ocd_id", "municipality")
+
+    type = graphene.String()
+    name = graphene.String()
+
+    def resolve_children(self, info, **kwargs):
+        return self.children.filter(type__type="sub_district")
+
+    def resolve_type(self, info, **kwargs):
+        return self.type.type
+
+
+class ProfileType(DjangoObjectType):
+    class Meta:
+        model = Profile
+        fields = (
+            "nickname",
+            "image",
+            "email",
+            "phone",
+            "language",
```

Comment:
we would need `"language"` and `"contact_method"` as proper enum fields:
```python
language = Language()
contact_method = ContactMethod()
```

`"language", "contact_method", "concepts_of_interest", "divisions_of_interest",` can be deleted from the fields, since declaring them directly on the object does not care about the `fields` attribute